test/hotspot/jtreg/compiler/vectorapi/VectorCompareWithImmTest.java

```diff
@@ -127,7 +127,7 @@ private static void assertArrayEquals(long[] a, boolean[] r, LongOp f) {
     }
 
     @Test
-    @IR(counts = { IRNode.VMASK_CMP_IMM_B_SVE, ">= 1" })
+    @IR(counts = { IRNode.VMASK_CMP_IMM_I_SVE, ">= 1" })
```

Comment:
Is this ok, or rather a downgrade?
Or maybe you should remove the `I` for int in the name?

Reply:
What would happen if I were to compare with a type that is larger? For example if I compared the bytes here with the value `1000`?

Reply:
> What would happen if I were to compare with a type that is larger? For example if I compared the bytes here with the value `1000`?

I suppose `javac` would complain since ByteVector.compare (https://github.com/openjdk/jdk/blob/master/src/jdk.incubator.vector/share/classes/jdk/incubator/vector/ByteVector.java#L2120) requires byte type.

Reply:
> Is this ok, or rather a downgrade? Or maybe you should remove the `I` for int in the name?

It's a downgrade. Since now on SVE platform, after matching they were generated to the same MachNode. `VMASK_CMP_IMM_I_SVE` is used to align with the rules' name on AArch64.